src/pet.js

```diff
@@ -0,0 +1,52 @@
+const maxFitness = 10
+const minHunger= 0
+
+class Pet {
+    constructor(name,age,hunger,fitness) {
+        this.name = name;
+        this.age=age;
+        this.hunger=hunger;
+        this.fitness=fitness; 
+    }
+    growUp() {
+        this.age += 1;
+        this.hunger+=5;
+        this.fitness-=3;
```

Comment:
Spaces either side of the operators make for easier reading
```suggestion
        this.hunger += 5;
        this.fitness -= 3;
```